src/core/client/ui/components/Modal/Modal.tsx

```diff
@@ -115,7 +118,11 @@ const Modal: FunctionComponent<Props> = ({
           >
             <div className={styles.alignContainer1}>
               <div className={styles.alignContainer2}>
-                <div className={styles.wrapper}>
+                <div
+                  className={styles.wrapper}
+                  role="presentation"
```

Comment:
Unsure if this is the correct role, but seems to be in-line with the role we used on the upper content wrapper that handles the escape key and scrolling.

Reply:
This seems correct. From my understanding, it just notes that the wrapper should not be mapped to a a11y interface (as it's just presentational).